### ISSUE TITLE
Prerender fails when using React component

### ISSUE THREAD

**Issue by thangngoc89** [developit/preact-cli#62]
This is yet another SSR bug :D 

Reproduce: 
- Include any 3rd party component and run build with prerender
- Error message: 

```
Template execution failed: Error: Cannot find module 'react'
  Error: Cannot find module 'react'
  
  - prerender.js:43 Function.Module._resolveFilename
    [frontend]/[preact-cli]/lib/lib/prerender.js:43:21
  
  - module.js:20 require
    internal/module.js:20:19
  
  - index.jsx:1 Object.<anonymous>
    /home/khoa/web/tooth/frontend/components/Search/index.jsx:1:1
  
  - node.js:144 loader
    [frontend]/[babel-register]/lib/node.js:144:5
  
  - node.js:154 Object.require.extensions.(anonymous function) [as .jsx]
    [frontend]/[babel-register]/lib/node.js:154:7
  
  - module.js:20 require
    internal/module.js:20:19
  
  - index.js:7 Object.<anonymous>
    /home/khoa/web/tooth/frontend/index.js:7:1
  
  - node.js:144 loader
    [frontend]/[babel-register]/lib/node.js:144:5
  
  - node.js:154 Object.require.extensions.(anonymous function) [as .js]
    [frontend]/[babel-register]/lib/node.js:154:7
```

**Comment by developit** [comment]
Ah weird, [module-alias](https://npm.im/module-alias) was actually never set up.  We need to add it to prerender.js.

**Comment by kurtextrem** [comment]
couldn't be a webpack build be used for the prerender? That way webpack's alias could be used and code functionality wouldn't somewhat get duplicated.

**Comment by developit** [comment]
Yes. Looks like @rkostrzewski is working on it because he's a genius.

**Comment by kurtextrem** [comment]
Awesome! LMK if you need / want help.

(and just for the overview - maybe add the SSR label to this issue and to the PR?)

**Comment by jangerhofer** [comment]
@developit: Do I understand correctly that the module-alias redirects `react` and `react-dom` includes to the corresponding `preact` packages, as described in the [`preact-compat`](https://github.com/developit/preact-compat) config?

I'm wondering because I'd like to use preact (and the convenience of the CLI!) with React packages, but `preact-compat` requires access to the `webpack.config`.

**Comment by developit** [comment]
Label added :)

@jangerhofer yes, that's what `module-alias` would be doing for us.  Webpack is already doing it for our browser build, but we are currently doing server rendering straight in Node.  Once we move the SSR to using Webpack, the aliases will be handled the same as they are in the browser.

**Comment by asci** [comment]
is there any workaround? `--prerender false` is not helping :(

**Comment by rkostrzewski** [comment]
@asci have you tried `--no-prerender`? If I recall correctly that's how to that with yargs

**Comment by asci** [comment]
@rkostrzewski oh, thanks. looks like it helps

**Comment by rkostrzewski** [comment]
@thangngoc89 this should work in `1.3.0`, could you verify?

**Comment by thangngoc89** [comment]
@rkostrzewski I'm forking preact-cli right now, gonna make some PR to make it easier for me to switch back

**Comment by reznord** [comment]
@thangngoc89 is it working on Rafal's fork?

**Comment by thangngoc89** [comment]
@reznord where is that fork? I have forked and check preact-cli in as a submodule inside my project.

**Comment by thangngoc89** [comment]
Fixed in 1.3.0
